lib/vmpooler/providers/vsphere.rb

```diff
@@ -428,7 +428,7 @@ def destroy_vm(pool_name, vm_name)
 
         def vm_ready?(_pool_name, vm_name)
           begin
-            open_socket(vm_name)
+            open_socket(vm_name, global_config[:config]['domain'])
```

Comment:
this looks like an unrelated change?

Reply:
Yes. I previously implemented the ability to specify a VM domain to use when opening a socket connection. Without this change that domain setting is never used. When interacting with a local instance of vmpooler in docker I found that I could not get VMs to reach the ready state without this addition. I could make this a separate commit if you think that makes more sense.